Added more Civil Engineering Imperial Units

osnippet opened the pull request.

neutrinoceros reviewed (approved):
Reviving #216 
For next time, know that it isn't necessary to open a brand new PR every time you want to push additional commits.

This looks fine to me.

osnippet commented:
As I mention in the original PR it has been a while since I added that changes. While transferring my changes I forgot how many files I have updated. When I saw all that test check errors I thought it was something major and it might require more times to fix so I closed that PR and turned out just some white space. Thank you for awesome work greatly appreciated.
